mesh/node.c

```diff
@@ -2692,6 +2691,88 @@ static struct l_dbus_message *amqp_identity_setter(struct l_dbus *dbus,
 	return NULL;
 }
 
+static bool is_opcode_valid(uint32_t opcode)
+{
+	if (opcode > 0xFFFFFFul || (opcode >> 16) == 0x7F)
```

Comment:
Why not use `mesh_model_opcode_get`?

Reply:
You mean `mesh_model_opcode_set`, right?